tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -99,7 +99,7 @@ Connect containers to multiple networks non-overlapping
 
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --net cross2-network2 --name cross2-container2 debian ping -c2 ${ip}
     Should Not Be Equal As Integers  ${rc}  0
-    Should contain  ${output}  Destination Port Unreachable
+    Should Contain  ${output}  2 packets transmitted, 0 packets received, 100% packet loss
```

Comment:
Looking at #4009 again, I see the test is not correct. I will add one to this PR.